subprojects/packagefiles/sdl2/meson.build

```diff
@@ -367,11 +368,24 @@ endif
 core_deps = [
     cc.find_library('m', required : false),
     cc.find_library('dl', required : false),
-    cc.find_library('iconv', required : false),
     dbus_dep,
     ibus_dep,
 ]
 
+if not opt_system_iconv.disabled()
+    iconv_dep = cc.find_library('iconv', required : opt_system_iconv)
+else
+    iconv_dep = fake_dep
+endif
+
+if cc.links('#include <iconv.h>\nint main() { iconv_open("", ""); }', name : 'iconv_open test')
```

Comment:
This does need to add iconv_dep as a dependency, because if iconv is standalone the links() check will fail to link without it.

Reply:
Yep, forgot to do that.